runtime/doc/filetype.txt

```diff
@@ -521,6 +521,8 @@ To use Nvim as a manpager: >
 man.vim will always attempt to reuse the closest man window (above/left) but
 otherwise create a split.
 
+The case sensitivity of completion is controlled by 'fileignorecase'.
```

Comment:
In `:h help-writing`, it says it's to show that it is an option, that's all. It's also done for commands, they are written as ``:filetype``. Just make's it easier to differentiate between them I guess.